zap/src/main/java/org/parosproxy/paros/core/scanner/Alert.java

```diff
@@ -691,6 +695,18 @@ public String toPluginXML(String urls) {
         }
         sb.append("  <sourceid>").append(source.getId()).append("</sourceid>\r\n");
 
+        if (tags != null && !tags.isEmpty()) {
```

Comment:
I don't think we should update this method, it was used for XML reports that were superseded with the new reports add-on (there are some add-ons making use of it, but if they want more information they should migrate to the new reports add-on).